chapters/functions.tex

```diff
@@ -1360,9 +1360,9 @@ \subsection{Using the Derivative Annotation}\label{using-the-derivative-annotati
 Assume that function \lstinline!f! takes a matrix and a scalar.
 Since the matrix argument is usually a parameter expression it is then
 useful to define the function as follows (the additional derivative =
-\lstinline!f_general_der! is optional and can be used when the derivative of
-the matrix or offset is non-zero). Note that \lstinline!f_der! must have
-\lstinline!zeroDerivative! for both \lstinline!y! and \lstinline!offset!, but \lstinline!f_general_der! shall not have
+\lstinline!fGeneralDer! is optional and can be used when the derivative of
+the matrix or offset is non-zero). Note that \lstinline!fDer! must have
+\lstinline!zeroDerivative! for both \lstinline!y! and \lstinline!offset!, but \lstinline!fGeneralDer! shall not have
 \lstinline!zeroDerivative! for either of them (it may \lstinline!zeroDerivative! for \lstinline!x_der!,
```

Comment:
It looks like there is a verb missing in "(it may \lstinline!zeroDerivative! for \lstinline!x_der!" Same problem as the comment above: the sentence was there before this PR. Should I open a separate issue?